hack/verify-zonal.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Jetstack Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Make the zonal cluster resource definiton from the regional cluster resource
+# definition. This helps to keep the two definitions the same, except for the
+# presence of the region or zone property.
+
+set -e
+
+mv gke/zonal.tf gke/zonal.tf.git
+./hack/make-zonal.sh
+DIFF=$(diff gke/zonal.tf gke/zonal.tf.git)
+if [ "$DIFF" != "" ]; then
+	exit 1
```

Comment:
May be worth printing `$DIFF` here before exiting?

Reply:
nit: perhaps add a 'file is out of date, run hack/make-zonal.sh and push up the changes'?